services-core/src/gas_price.rs

```diff
@@ -5,13 +5,15 @@ pub use self::gas_station::GnosisSafeGasStation;
 use crate::{contracts::Web3, http::HttpFactory};
 use anyhow::Result;
 use ethcontract::U256;
-use std::sync::Arc;
+use std::{sync::Arc, time::Duration};
 
 #[cfg_attr(test, mockall::automock)]
 #[async_trait::async_trait]
 pub trait GasPriceEstimating {
-    /// Retrieves gas prices from the Gnosis Safe Relay api.
-    async fn estimate_gas_price(&self) -> Result<U256>;
+    /// Estimate the gas price for a transaction to be mined "quickly".
+    async fn estimate_simple(&self) -> Result<U256>;
```

Comment:
I really don't like the name `_simple` here as it doesn't give any information about what it does. Maybe we can just call it estimate and the other one `estimate_with_limit` or call this one `estimate_fast_execution`?

Reply:
Both functions do the same thing but one does it based on more parameters. That's why I picked "simple" and implicitly "complex". Could also be coarse and fine grained. But i have no strong feelings about the name so I'll go with your suggestion "estimate" and "estimate_with_limits".